src/mixins/amalgamation-mixin.ts

```diff
@@ -28,6 +28,7 @@ export default class AmalgamationMixin extends Vue {
     this.notInGoodStanding,
     this.limitedRestoration,
     this.futureEffectiveFiling,
+    this.pendingDissolutionFiling,
```

Comment:
Note that we check future-effective before pending-resolution, per Yui:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/a781e1a2-471f-4964-a3ba-5ec6c2767b68)